apps/meteor/app/livechat/server/api/v1/message.ts

```diff
@@ -0,0 +1,293 @@
+import { Meteor } from 'meteor/meteor';
```

Comment:
We're importing Meteor in this file just for its Meteor Wrapper, so can we change it to `Error` to remove this dependency altogether?